src/hotspot/share/compiler/compilationFailureInfo.cpp

```diff
@@ -33,12 +33,12 @@
 #include "compiler/compilationFailureInfo.hpp"
 #include "compiler/compileTask.hpp"
 #ifdef COMPILER2
-#include "opto/node.hpp"
 #include "opto/compile.hpp"
+#include "opto/node.hpp"
```

Comment:
Here the entire:
```
#ifdef COMPILER2
#include "opto/compile.hpp"
#include "opto/node.hpp"
#endif
```
block should be last. I don't know if your tool should try to fix this.